Tests/SonarScanner.MSBuild.Test/ArgumentProcessorTests.cs

```diff
@@ -22,364 +22,339 @@
 using System.IO;
 using FluentAssertions;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
-using SonarScanner.MSBuild;
 using SonarScanner.MSBuild.Common;
 using TestUtilities;
 
-namespace SonarScanner.MSBuild.Test
+namespace SonarScanner.MSBuild.Test;
```

Comment:
Together with this, we can also remove all the `#region`s within these test files

Reply:
I have removed the `Public methods` region.

I have also removed the `Argument definitions` region. That, however, called for a bit of refactoring on the `Descriptors` initialization since the region had the advantage of keeping close `Descriptors` declaration and its initialization, and I wanted to keep that.